ci/infra/azure/terraform.tfvars.example

```diff
@@ -0,0 +1,103 @@
+# AZURE location
+# A list of location names can be found here:
+# https://azure.microsoft.com/en-in/global-infrastructure/locations/
+# https://azure.microsoft.com/en-us/global-infrastructure/regions/
+# https://docs.microsoft.com/en-us/azure/availability-zones/az-region
+# azure_location = "West Europe"
+
+# Enable zone for available regions
+# EXAMPLE:
+# enable_zone = true
+
+# List of Availability Zones (e.g. `["1", "2", "3"]`)
+# EXAMPLE:
+# azure_availability_zones = ["1", "2", "3"]
+
+# CIRD blocks for virtual network
+# EXAMPLE:
+# cidr_block = "10.1.0.0/16"
+
+# CIDR blocks for the bastion host subnet
+# EXAMPLE:
+# bastionhost_subnet_cidr = "10.1.1.0/24"
+
+# Private subnet of virtual_network
+# EXAMPLE:
+# private_subnet_cidr = "10.1.4.0/24"
+
+# Identifier to make all your resources unique and avoid clashes with other users of this terraform project
+# EXAMPLE:
+# stack_name = "my-sles-cluster"
```

Comment:
Suggest to uncomment or add required configuration items (without `#`). 

Ref: 
* https://github.com/SUSE/skuba/blob/master/ci/infra/openstack/terraform.tfvars.example, 
* https://github.com/SUSE/skuba/blob/master/ci/infra/vmware/terraform.tfvars.example

As example:
```suggestion
stack_name = "my-sles-cluster"
```

Reply:
Please also go through the other configuration items to uncomment or add what is required.